src/intro/introduction.md

```diff
@@ -3,6 +3,10 @@
 Welcome to The Embedded Rust Book: An introductory book about using the Rust
 Programming Language on "Bare Metal" embedded systems, such as Microcontrollers.
 
+## Who Embedded Rust is For
+Embedded rust is for everyone who wants to do embedded programming backed by the higher-level concepts and safety guarantees the rust language provides.
```

Comment:
I agree with @thejpster: Please go through the instances where you used "rust" and replace them with "Rust".